src/java/org/apache/cassandra/db/guardrails/GuardrailsMBean.java

```diff
@@ -553,4 +553,21 @@
      */
     void setMinimumReplicationFactorThreshold (int warn, int fail);
 
+    /**
+     * @return The threshold to fail when replication factor is greater threshold.
+     */
+    int getMaximumReplicationFactorWarnThreshold();
+
+    /**
+     * @return The threshold to fail when replication factor is greater threshold.
```

Comment:
```suggestion
     * @return The threshold to fail when replication factor is greater than threshold.
```
This also applies to the equivalent `getMinimumReplicationFactorFailThreshold`.

Reply:
As mentioned above, I would also apply this fix to `getMinimumReplicationFactorFailThreshold`